surface/src/on_nurbs/on_nurbs.cmake

```diff
@@ -1,4 +1,24 @@
 
+message(STATUS "EIGEN_VERSION: ${EIGEN_VERSION}")
+message(STATUS "EIGEN_WORLD_VERSION: ${EIGEN_WORLD_VERSION}")
+message(STATUS "EIGEN_MAJOR_VERSION: ${EIGEN_MAJOR_VERSION}")
+message(STATUS "EIGEN_MINOR_VERSION: ${EIGEN_MINOR_VERSION}")
+
+#set(PREPROCESSOR_EIGEN_VERSION "-DEIGENVERSION_GEQUAL_3_2")
```

Comment:
Sorry, maybe I misunderstood the purpose of this pull request (no description provided). From reading the source changes I assumed that you are adding an alternative implementation, which relies on a modern version of Eigen. If it is available, then the new code could/should be used, otherwise fallback to the old code. Therefore, I was puzzled by the version test commented out completely. If there is no way to switch the new code on, then does it make sense to contribute it right now?